tests/yaml/co2_mass_yaml_example.yml

```diff
@@ -0,0 +1,14 @@
+co2_mass_settings:
+  init_source: /scratch/fmu_for_ccs/ause/test_data_for_NR/data_sets/01_pflotran_test_1.6sv5/realization-0/iter-0/pflotran/model/SMEAHEIA_PFT-0.INIT
+  unrst_source: /scratch/fmu_for_ccs/ause/test_data_for_NR/data_sets/01_pflotran_test_1.6sv5/realization-0/iter-0/pflotran/model/SMEAHEIA_PFT-0.UNRST
+input:
+  grid: /scratch/fmu_for_ccs/ause/test_data_for_NR/data_sets/01_pflotran_test_1.6sv5/realization-0/iter-0/pflotran/model/SMEAHEIA_PFT-0.EGRID
+mapsettings:
+  ncol: 560
+  nrow: 600
+  xinc: 40.0
+  xori: 548770
+  yinc: 40.0
+  yori: 6695125
+output:
+  mapfolder: /scratch/fmu_for_ccs/ause/test_data_for_NR/data_sets/01_pflotran_test_1.6sv5/realization-0/iter-0/share/results/maps

```

Comment:
Can remove from commit, since it is not used in the repo? Can at some point make test cases similar to time migration and aggregate (but would need a synethetic data set with all required properties...)

Reply:
Completely agree. I had forgotten this one was there :)